sirepo/auth/__init__.py

```diff
@@ -160,10 +169,14 @@ def init_apis(*args, **kwargs):
     visible_methods = tuple(sorted(visible_methods))
     non_guest_methods = tuple(m for m in visible_methods if m != METHOD_GUEST)
     cookie.auth_hook_from_header = _auth_hook_from_header
+    s = simulation_db.SCHEMA_COMMON.common.constants.paymentPlans.keys()
```

Comment:
This doesn't quite work right. keys returns an iterator so the `{s}` will print the iterator, not the value. I think in this case you want:

```py
s = list(simulation_db.SCHEMA_COMMON.common.constants.paymentPlans.keys())
assert sorted(s) == sorted(_ALL_PAYMENT_PLANS), \
```

We want to make sure the two lists are identical. Could just test the length, but this is just as easy since it is as startup

Reply:
https://github.com/radiasoft/sirepo/pull/2711/commits/05bde41c8e65d6cbc52101463c4a1a6418c2356b